admin/formatter/class-term-metabox-formatter.php

```diff
@@ -53,14 +53,15 @@ public function get_values() {
 		// Todo: a column needs to be added on the termpages to add a filter for the keyword, so this can be used in the focus keyphrase doubles.
 		if ( is_object( $this->term ) && property_exists( $this->term, 'taxonomy' ) ) {
 			$values = [
-				'search_url'          => $this->search_url(),
-				'post_edit_url'       => $this->edit_url(),
-				'base_url'            => $this->base_url_for_js(),
-				'taxonomy'            => $this->term->taxonomy,
-				'keyword_usage'       => $this->get_focus_keyword_usage(),
-				'title_template'      => $this->get_title_template(),
-				'metadesc_template'   => $this->get_metadesc_template(),
-				'first_content_image' => $this->get_image_url(),
+				'search_url'               => $this->search_url(),
+				'post_edit_url'            => $this->edit_url(),
+				'base_url'                 => $this->base_url_for_js(),
+				'taxonomy'                 => $this->term->taxonomy,
+				'keyword_usage'            => $this->get_focus_keyword_usage(),
+				'title_template'           => $this->get_title_template(),
+				'metadesc_template'        => $this->get_metadesc_template(),
+				'first_content_image'      => $this->get_image_url(),
+				'semrushIntegrationActive' => 0,
```

Comment:
The format used to name the key, is completely different of that of the other keys. Is there a specific reason for this? If not, please change this to snake case.

Reply:
Noting this is the existing key name in `admin/formatter/class-metabox-formatter.php` which is similar to other key names related to "active features" e.g. `contentAnalysisActive`, `keywordAnalysisActive`, `cornerstoneActive`. No objections to change it but then I'd say that for consistency they should all be changed. Will check better tomorrow 🙂

Reply:
@afercia @jcomack has a conclusion yet been drawn?

Reply:
Yes, the current approach is fine and thus this PR can move on through the process.